pub_sub/go/http/checkout/app.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const PUBSUB_NAME = "order_pub_sub"
+const PUBSUB_TOPIC = "orders"
+
+func main() {
+	daprHost := "http://localhost"
+	if value, ok := os.LookupEnv("DAPR_HOST"); ok {
+		daprHost = value
+	}
+	daprHttpPost := "3500"
+	if value, ok := os.LookupEnv("DAPR_HTTP_PORT"); ok {
+		daprHttpPost = value
+	}
+	for i := 1; i <= 10; i++ {
+		order := `{"orderId":` + strconv.Itoa(i) + `}`
```

Comment:
Rather than string concat with + operator, I believe fmt.Sprintf is more used, and performant.  @pkedy ?

```suggestion
		order := fmt.Sprintf(`{"orderId": %d}`, i)
```